README.md

```diff
@@ -0,0 +1,22 @@
+# ROUTE-METRICS-DISPLAY
+
+`route-metrics-display` was created to visualize data created from `route-metrics`.
+It reads a `route-metrics.log` file and displays all the information provided as charts on a webpage.
+The file can be provided either by uploading it or through it's local filepath, in which case `route-metrics-display` is gonna continuously watch that file for changes.
+
+# USAGE
+
+Start your server with `route-metrics` first (https://github.com/Contrast-Security-OSS/node-route-metrics for instructions).
+
+After that, execute `git clone https://github.com/Contrast-Security-OSS/route-metrics-display` to clone the current repository.
+Once that's done, do `cd route-metrics-display` to visit the new folder and execute `npm i` to install all the dependencies `route-metrics-display` has.
```

Comment:
I think `npm ci` would be better

Reply:
`npm install` is the standard way to tell people to install code; i'm thinking that telling people how to use `npm` is outside the scope of this. is there a compelling reason to tell them to use `npm ci`?